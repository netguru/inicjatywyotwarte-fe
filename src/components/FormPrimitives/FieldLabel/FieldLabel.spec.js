import React from 'react'
import Label from './FieldLabel'
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

describe('Label', () => {
  test('renders styled.label', () => {
    const wrapper = shallow(<Label/>);
    expect(wrapper.find('label').length).toEqual(1);
  });

  test('match snapshot', () => {
    const wrapper = shallow(<Label/>);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});


