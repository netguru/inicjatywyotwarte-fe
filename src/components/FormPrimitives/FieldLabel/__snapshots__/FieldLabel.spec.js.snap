// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Label match snapshot 1`] = `
<label
  className="sc-AxjAm jywRGM"
/>
`;
